docs/configuration.md

```diff
@@ -77,6 +78,23 @@ spec:
       - name: allow-ssh
 ```
 
+## Application Credentials
```

Comment:
This should be noted in `OpenStack credential`. Because application-credential-id and application-credential-secret are included in `OPENSTACK_CLOUD_YAML_B64` and `OPENSTACK_CLOUD_PROVIDER_CONF_B64`. And generally "credential" means information such as user id and password or credential id and secret to authenticate.